ui/apps/platform/src/Containers/Violations/ViolationsTablePage.tsx

```diff
@@ -285,6 +291,7 @@ function ViolationsTablePage(): ReactElement {
                             onFilterChange={setSearchFilter}
                             onSearch={onSearch}
                             additionalContextFilter={additionalContextFilter}
+                            hasActiveViolations={selectedViolationStateTab === 'ACTIVE'}
```

Comment:
An observation, not necessarily a change request:
* In context of use, I initially read `has` as it has active violations, but in testing with incomplete workview filter, it did not have any.
* However, instead of renaming the boolean prop, what are pro and con for `selectedViolationStateTab` to be the prop, in case conditions at lower level ever need to distinguish among all tabs?

Reply:
I considered passing `selectedViolationStateTab`, but I felt that the component shouldn't know the tab state because of the separation of concerns. It would have been nice if I could do the logic at this level and pass a prop like `hasTableActions`, but I realized the table component needs to use `hasWriteAccessForAlert` and `hasWriteAccessForExcludeDeploymentsFromPolicy` individually too. What do you think about pulling those up and passing them as props?

Reply:
Aha, then disregard my comment.